### ISSUE TITLE
regex containing backslashes at sublime-keymap is not highlighted correctly

### ISSUE THREAD

**Issue by randy3k** [SublimeText/PackageDev#130]
<img width="895" alt="screen shot 2017-08-24 at 1 39 03 am" src="https://user-images.githubusercontent.com/1690993/29651135-12ccbe1a-886d-11e7-9037-516f2a87b215.png">

```
{ "key": "following_text", "operator": "regex_contains", "operand": "^[0-9a-zA-Z\\\\\\(\\)\\[\\]{}]", "match_all": true }
```

**Comment by r-stein** [comment]
I think this uses the [old escaped regex syntax](https://github.com/SublimeText/PackageDev/blob/v3.0.0-beta.3/Package/Regular%20Expression%20(Escaped).JSON-tmLanguage).

**Comment by FichteFoll** [comment]
Yes indeed, we don't have a new one yet. I deem it to be necessary before #67 can be marked off so that we don't have any dependencies on the old syntax definitions anymore.
